### ISSUE TITLE
SSH management

### ISSUE THREAD

**Issue by tomberek** [cgzones/easy-ca#4]
With the structure present, it would be pretty easy to add SSH key management (vanilla and CA-style). Thoughts?

**Comment by cgzones** [comment]
I am currently not using some kind of SSH key management (yet).
What functionality do you expect? Something similar to [this](https://github.com/cloudtools/ssh-ca) project?
* creating CA
* create signed host certificate
* create signed user certificate with adjustable lifetime
* ...

**Comment by tomberek** [comment]
Yep. It ends up being similar, but simpler, than what you do here for x509. Fewer options.

**Comment by tomberek** [comment]
#6 starts to address this. Will add SSH CA support next.
